src/components/dao/settings-general.vue

```diff
@@ -39,9 +39,9 @@ widget(title='General' titleImage='/svg/file-checkmark.svg' :bar='true').q-pa-no
                 lazy-rules
                 outlined
                 placeholder="Paste the URL address here"
-                ref="name"
+                ref="title"
```

Comment:
I may be wrong, but I've dealt with this issue before, and shouldn't the 'name' parameter be changed instead of 'title'?

Reply:
I added to be called name as it represent the name, inside save function i am remapping name -> daoTitle